src/native-stack/utils/useScreenInfo.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+
+import ScreenInfoContext from './ScreenInfoContext';
+
+export default function useScreenInfo() {
+  const screenInfo = React.useContext(ScreenInfoContext);
+
+  if (screenInfo === undefined) {
+    throw new Error(
+      "Couldn't find information about the screen. Are you inside a screen in a navigator?"
```

Comment:
```suggestion
      "Couldn't find information about the screen. Make sure that the hook is being used inside screen contained in navigator"
```

Reply:
Since this hook has been removed, I'm resolving it. Context: https://github.com/software-mansion/react-native-screens/pull/1915/commits/b2d876ac20a3972688451fdf3f9b62c12086f6c2.